src/components/viewPaginationRow/viewPaginationRow.js

```diff
@@ -1,89 +1,46 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { PaginationRow, PAGINATION_VIEW } from 'patternfly-react';
+import { Pagination } from '@patternfly/react-core';
```

Comment:
required:
import the PaginationVariant and prefer using that over any "strings", ie `PaginationVariant.bottom`. saves us future bits when PF decides to alter those enums/strings